### PR TITLE
fix: release script fixed to add correct build flags

### LINKED ISSUE #322: v0.7.0 says  DB1000n [Version: v0.0.1]
I assume it's left over hardcoded version value or somewhere the code has not been changed properly

Screenshot 2022-03-13 at 13 14 20

Reproduced via executables on MacOS
- for Intel CPU https://github.com/Arriven/db1000n/releases/download/v0.7.0/db1000n-v0.7.0-darwin-amd64.tar.gz
- for M1 CPU https://github.com/Arriven/db1000n/releases/download/v0.7.0/db1000n-v0.7.0-darwin-arm64.tar.gz

cc/ @Arriven ; @bitshape

### PR DESCRIPTION
**Pull request by arfgdev**
# Description

release binaries github action fixed
Fixes #322

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Arriven** [comment]
Already fixed, there was a caveat of making make build output .exe in order to comply with the action (just one more point in my "why I don't like windows" list). Changing the version name to development seems like a nice idea, need to test it out though

File: src/utils/ota/version.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package ota
 
 var (
 	// Version is a releae version embedded into the app
-	Version = "v0.0.1"
+	Version = "development"
```
Comment (Arriven) [inline src/utils/ota/version.go:5]: @arfgdev can you ensure OTA updates functionality works well with this change?

Comment (arfgdev) [reply src/utils/ota/version.go:5]: +

Comment (arfgdev) [reply src/utils/ota/version.go:5]: it does not work 😅
`Binary version validation failed: Invalid character(s) found in major number "development"`

Comment (sivillakonski) [reply src/utils/ota/version.go:5]: It won't work because it doesn't follow the semver semantics

Comment (Arriven) [reply src/utils/ota/version.go:5]: I'll close the PR then

**Pull request closed without merging**